resources/views/partials/forms/edit/eol_date.blade.php

```diff
@@ -3,7 +3,7 @@
     <label for="asset_eol_date" class="col-md-3 control-label">{{ trans('admin/hardware/form.eol_date') }}</label>
     <div class="input-group col-md-4">
         <div class="input-group date" data-provide="datepicker" data-date-clear-btn="true" data-date-format="yyyy-mm-dd"  data-autoclose="true">
-            <input type="text" class="form-control" placeholder="{{ trans('general.select_date') }}" name="asset_eol_date" id="asset_eol_date" readonly value="{{  old('asset_eol_date', $item->asset_eol_date ?? $item->present()->eol_date() ?? '')  }}"  style="background-color:inherit">
+            <input type="text" class="form-control" placeholder="{{ trans('general.select_date') }}" name="asset_eol_date" id="asset_eol_date" readonly value="{{  old('asset_eol_date', $item->asset_eol_date->format('Y-m-d') ?? $item->present()->eol_date() ?? '')  }}"  style="background-color:inherit">
```

Comment:
This fails when trying to create an asset.

```
Call to a member function format() on null
```

Changed it to this seems to work:
```
optional($item->asset_eol_date)->format('Y-m-d')
```